cognite/v06/sequences.py

```diff
@@ -0,0 +1,107 @@
+# -*- coding: utf-8 -*-
+"""Sequences Module
+
+This module mirrors the Sequences API.
+
+https://doc.cognitedata.com/api/0.6/#tag/Sequences
+"""
+import json
+from typing import List
+
+from cognite import _utils, config
+from cognite.v06.dto import Sequence, Column
+
+
+def post_sequences(
+        sequences: List[Sequence],
+        **kwargs
+):
+    """Create a new time series.
+
+    Args:
+        sequences (list[v06.dto.Sequence]): List of sequence data transfer objects to create.
+
+    Keyword Args:
+        api_key (str): Your api-key.
+        project (str): Project name.
+
+    Returns:
+        The created sequence
+    """
+
+    api_key, project = config.get_config_variables(kwargs.get("api_key"), kwargs.get("project"))
+    url = config.get_base_url() + "/api/0.6/projects/{}/sequences".format(project)
+
+    body = {"items": [sequence.__dict__ for sequence in sequences]}
+
+    headers = {"api-key": api_key, "content-type": "application/json", "accept": "application/json"}
+
+    res = _utils.post_request(url, body=body, headers=headers)
+
+    json_response = json.loads(res.text)
+    the_sequence: dict = json_response['data']['items'][0]
+
+    return Sequence(
+        id=the_sequence['id'],
```

Comment:
I would consider making a .from_JSON() method in the Sequence class, so you don't have to duplicate this mapping every time.

Reply:
Done in https://github.com/cognitedata/cognite-sdk-python/pull/150/commits/4d559088e154387d71df2b68e7c0cb9cf47fb5ff